### DISCUSSION TITLE
WABC-DT1

### DISCUSSION THREAD

**Discussion by chammer977** [iptv-org/community#891] [Q&A]
### Channel ID

WABCDT1.us

### Notes (optional)

Since links like http://v-ny.theus6tv.duckdns.org/hls/7.1/playlist.m3u8 have gone down, are there any working ABC, CBS or NBC main EAST coast stations working? I can only find the News stations on the playlist and I'd like to see full network programming.

Thanks

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/299c979aed4d48aa5b3ec4a05458725957af64cf/streams/us_local.m3u#L538-L539

**Comment by chammer977** [comment]
Hi. That's the channel that only shows the local news, not the full WABC-TV channel. Similar to iptv-org/iptv#8935, as this is only the news part.

**Reply by Spainguy77** [reply]
Did you find a solution? Have been looking myself but no luck so far

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19948
